### PR TITLE
Tutorial for closure_phase_bias.py

### PR DESCRIPTION
**Pull request by yjzhenglamarmota**
Tutorial for closure_phase_bias.py (https://github.com/insarlab/MintPy/pull/765).

Data can be downloaded from https://doi.org/10.5281/zenodo.6558341

### CODE REVIEW

**Comment by yunjunz** [comment]
Very nice notebook demonstration @yjzhenglamarmota! I only have a few minor comments above. 

Could you also add a link to this notebook to the README.md file under `4. Custom applications` for easy navigation?

**Comment by yjzhenglamarmota** [comment]
> Very nice notebook demonstration @yjzhenglamarmota! I only have a few minor comments above.
> 
> Could you also add a link to this notebook to the README.md file under `4. Custom applications` for easy navigation?

I have committed suggested changes.

**Comment by yunjunz** [comment]
> hi dear yjzhenglamarmota , can you guide me please, on how can I install package closure_phase_bias.py?

@sisi-ali The closure phase bias PR (#765) is not merged yet, which means it's not ready to be used. 

Please try to not post the same comment in multiple places. Comments here are relevant and welcomed.

**Comment by yjzhenglamarmota** [comment]
I have updated the notebook to keep up with the new and hopefully finalized version in MintPy.

File: README.md [hunk 1]
```diff
@@ -43,6 +43,8 @@ List of examples of how to build customized application using mintpy modules or
    - Plot GPS as quiver on top of InSAR data: [nbviewer](https://nbviewer.jupyter.org/github/insarlab/MintPy-tutorial/blob/main/applications/plot_gps_quiver.ipynb)
    - Average velocity estimation demonstration: [nbviewer](https://nbviewer.jupyter.org/github/insarlab/MintPy-tutorial/blob/main/applications/ts2vel.ipynb)
    - Coherence Change Detection: [nbviewer](https://nbviewer.jupyter.org/github/insarlab/MintPy-tutorial/blob/main/applications/coherence_change_detection.ipynb)
+   - Mask/ estimate/ correct phase non-closure related bias: nbviewer](https://nbviewer.jupyter.org/github/insarlab/MintPy-tutorial/blob/main/applications/tutorial_biasCorrection.ipynb)
+
```
Comment (yunjunz) [inline README.md:47]: How about naming this notebook file as `closure_phase_bias.ipynb`?

**Comment by yunjunz** [comment]
<p>Could you add an explanation for the <code>quick_estimate</code> like, based on..., or assuming ....? </p>

---

 <a href='https://app.reviewnb.com/insarlab/MintPy-tutorial/pull/40/discussion/'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Comment by sisi-ali** [comment]
Hi dear
I'm using ISCE software and I can't download DEM.
How can I download DEM ?
Could you please guide me?
I have this Error()
s a problem in retrieving the file  - WARNING  - There was
http://e4ftl01.cr.usgs.gov/MEASURES/SRTMGL1.
http://e4ftl01.cr.usgs.gov/MEASURES/SRTMGL1.
003/2000.02.11/N30E049.SRTMGL1.hgt.zip. Exception  Could not create a
stitched DEM. Some tiles are missing





‫‪review-notebook-app[bot]‬‏ ***@***.***‬‏> در تاریخ
چهارشنبه ۲۷ ژوئیه ۲۰۲۲ ساعت ۵:۳۱ نوشت:‬

**Comment by sisi-ali** [comment]
<p>Hi dear&nbsp;</p><p>I'm using&nbsp;ISCE software and I can't&nbsp;download DEM.&nbsp;</p><p>How can I download DEM ?&nbsp;</p><p>Could you please guide&nbsp;me?</p><p>I have this Error()</p><p>s a problem in retrieving the file&nbsp;- WARNING&nbsp;- There was&nbsp;&nbsp;<a href="http://e4ftl01.cr.usgs.gov/MEASURES/SRTMGL1" target="_blank">http://e4ftl01.cr.usgs.gov/MEASURES/SRTMGL1</a>.</p><p><a href="http://e4ftl01.cr.usgs.gov/MEASURES/SRTMGL1" target="_blank">http://e4ftl01.cr.usgs.gov/MEASURES/SRTMGL1</a>. 003/2000.02.11/N30E049.SRTMGL1.hgt.zip. Exception&nbsp;Could not create a stitched DEM. Some tiles are missing</p>

---

 <a href='https://app.reviewnb.com/insarlab/MintPy-tutorial/pull/40/discussion/'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Comment by yunjunz** [comment]
@sisi-ali Check here: https://github.com/isce-framework/isce2/issues/458#issuecomment-1191886597

**Comment by yjzhenglamarmota** [comment]
@yunjunz  Updated notebook

**Comment by sisi-ali** [comment]
hi dear,
I can not download DEM yet.
my study area is 30 33 47 50

‫‪Yujie Zheng‬‏ ***@***.***‬‏> در تاریخ سه‌شنبه ۲ اوت ۲۰۲۲
ساعت ۲۳:۳۲ نوشت:‬

**Review by yunjunz — APPROVED** [review]
Looks all good to me. Thank you @yjzhenglamarmota for this super useful notebook!!

**Comment by sisi-ali** [comment]
hi
I work with MintPy software and I want to get phase_clouser_bias, but it is
not possible, I use this code (
https://github.com/insarlab/MintPy/blob/main/mintpy/closure_phase_bias.py)
Thank you for guiding me

‫‪sedigheh alihasani‬‏ ***@***.***‬‏> در تاریخ
پنجشنبه ۴ اوت ۲۰۲۲ ساعت ۱۰:۳۳ نوشت:‬

**Comment by yunjunz** [comment]
@sisi-ali please keep the same posting to one issue/discussion only.

**Pull request merged**